### PR TITLE
AGENT-974: Add integration test for agent minimal ISO

### PR DESCRIPTION
**Pull request by bfournie**
Add some new integration functions to check minimum and maximum ISO file size along with a new integration test for the agent minimal ISO support.

### CODE REVIEW

File: internal/tshelpers/custom_commands.go [hunk 1]
```diff
@@ -302,6 +305,50 @@ func ExpandFile(ts *testscript.TestScript, neg bool, args []string) {
...
+}
+
+// IsoSizeMax `isoPath` `size` checks if the specified ISO is smaller
+// than the specified number of bytes.
+func IsoSizeMax(ts *testscript.TestScript, neg bool, args []string) {
+	if len(args) != 2 {
+		ts.Fatalf("usage: IsoSizeMa isoPath size")
+	}
+
+	workDir := ts.Getenv("WORK")
```
Comment (rwsu) [inline internal/tshelpers/custom_commands.go:337]: I think we could eliminate some duplicate code by having a IsoSize function that just returns the size of the file and then have IsoSizeMin and IsoSizeMax call it. WDYT?

File: internal/tshelpers/custom_commands.go [hunk 2]
```diff
@@ -31,6 +32,8 @@ import (
 // 						   | during the comparison.																   |
 // IsoFileCmpRegEx 		   | checks that file context extracted directly from the ISO matches the content of the   | isoFileCmpRegEx node.x86_64.iso /EFI/redhat/grub.cfg expected/grub.cfg
 //						   | local file, by applying a regex comparison.										   |
+// IsoSizeMin 		   | checks that the ISO is greater than a minimum number of bytes                         | isoSizeMin agent.x86_64.iso 100000000
```
Comment (rwsu) [inline internal/tshelpers/custom_commands.go:35]: Could we specify Min and Max by megabytes? It would be easier to comprehend.

**Review by rwsu — CHANGES_REQUESTED** [review]

File: internal/tshelpers/custom_commands.go (hunk 2 above)
Comment (bfournie) [reply internal/tshelpers/custom_commands.go]: Good idea. Changed.

File: internal/tshelpers/custom_commands.go (hunk 1 above)
Comment (bfournie) [reply internal/tshelpers/custom_commands.go]: Changed to use function.

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**